lib/DraggableCore.es6

```diff
@@ -294,6 +302,7 @@ export default class DraggableCore extends React.Component {
     // Reset the el.
     this.setState({
       dragging: false,
+      scrolling: true,
```

Comment:
This is going to cause us to always assign a 'touchmove' handler if the draggable has been dragged more than once.